### ISSUE TITLE
Work out best structure for the DoES Liverpool company

### ISSUE THREAD

**Issue by johnmckerrell** [DoESLiverpool/somebody-should#321]
With the move it might make sense to restructure how DoES Liverpool is set up. It is currently a CIC limited by share but we believe there might be better options:
- Limited by guarantee with more clear indication that we're a not for profit
  - May allow us to successfully claim discretionary rates relief
- Charity
  - Should give us greater scope for mandatory rates relief

It is likely that we will need to register for VAT as the new rent will be VATable, we will try to do this in such a way so that this impacts the community as little as possible.

We may be able to reduce the amount of corporation tax that we're paying by considering most of our income as mutual trading. We would need a more official membership scheme that members would join but would then not have to pay corporation tax on that income. Any income from non-members would still be taxable. This may offset VAT but again we need to make sure to reduce any negative impacts.

**Comment by mwnw00359** [comment]
LCVS can help here, might be worth converting the current CIC to a Charitable structure, perhaps as a group company, the main charity and it owning trading companies in order to keep things separate,protecting its assets from the trading side. See https://www.gov.uk/guidance/change-your-charity-structure

**Comment by stevesparrow** [comment]
Thanks, I met with LCVS at the end of July, and we agreed that a charity could work, and looked at some different examples of charitable aims.  I'm planning to put together charitable aims for DoES, and run them back by the directors.  We're interested in this (and a holding company) as long as we can use charitable aims that let us be all we need to be.  If anyone wants to offer help, happy to do it together.

**Comment by mwnw00359** [comment]
Hey Steve, 

Sure, happy to help, I did a governance review and re-did quite a lot of the constitution at Liverpool Pride when I was trustee there, in which I met with Richard from LCVS who was really helpful in getting the legals right etc.

Personally, I would start off with the vision and strategy for doES and push it down from there, i.e where does it want to be in 3/5 years and how will it do it and what will it feel like to members/users. You then find the appropriate structure that fits the strategy rather than create the structure and try to get it to fit the strategy,  either way you will need to create a sustainable organisation who's objects are pointed at the community, either being a developer of space (landlord) or more than that but clearly defined.

**Comment by stevesparrow** [comment]
Thanks Matt,

It sounds like we're in a similar place, in that, this is what we've been doing.  At the moment, we're exploring to what extent a charity structure can easily fit who we are, which is to say, whether all of who we are  / what we do and want to do can fit within either charitable objectives and a separate wholly owned company.  My background is in charities and my hunch is that it can, but we need to do the groundwork to be sure.  When Castle street dropped through, I paused and need to pick it back up!

Potentially, it could save us a whole bunch of money in rates etc, which would be excellent.  I can share some stuff with you once I've written it up? :)

Thanks

Steve

**Comment by mwnw00359** [comment]
Yes please do, my point last time is that doES should do it now rather than wait till a building comes along, for many reasons including the signing of a lease in a new company might raise security issues (bigger deposit's required etc) or add complications at the last min, I would actually suggest doing the opposite (than how some have explained it to me), get the new structure up and running and finalised BEFORE looking around at buildings so you are going in prepared. The rates is a good point and there could be considerable savings for doES activities but beware LCC's polices and advice over the phone, you may find that rates are payable on the commercial areas(renting desks, rooms, meeting room etc) as it very much depends on their view of the new doES , so I suggest a recorded meeting with them to communicate the doES business plan, then let them tell you the implications and then factor those into the pricing before moving forward on any building so you know your costs and liabilities in advance.

**Comment by stevesparrow** [comment]
I struggled with this somewhat, but it's been superseded by the vat on rent question, where a mixed approach looks best.  Closing for now.

_Sent from my Asus P027 using [FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github)_

**Comment by amcewen** [comment]
We are still looking at switching to limited by guarantee though, aren't we?

**Comment by johnmckerrell** [comment]
Some feedback on Limited by Guarantee, this page suggests there is no "conversion" process:

http://www.companylawclub.co.uk/converting-a-company-from-one-type-to-another

While in theory setting up a new company and transferring assets shouldn't really be all that tricky, talking to the accountant he did point out that CIC's are just a bit more difficult because everything has to be done with actual paperwork rather than just online. Accountant also suggested we might amend the articles of association to specify that directors do not take dividends and then if we were applying for funding we would specify this in our application.

**Comment by johnmckerrell** [comment]
I don't know for sure but when we do this I imagine we might have to create a new bank account. Co-op being such a pain in general I was wondering if this would be a good time to move away from them. I just ready about Unity Bank which might be a decent alternative:

https://www.unity.co.uk

**Comment by DoESsean** [comment]
Are we still looking at changing anything about the structure? Is Limited by Guarantee still a legit option? Or are we happy how we are?

**Comment by johnmckerrell** [comment]
Things to find out:
* Would we actually have to get another bank account with a different number or could we keep using the existing one?
* How much would the accountant charge us and how much effort would it be to sort out?
* What would we have to tell our members?

**Comment by jackie1050** [comment]
I've spoken to my sister, who might be able to help. I've given her the info from this email.

**Comment by jackie1050** [comment]
I've emailed our accountant with the questions in @johnmckerrell comment from 17 September

**Comment by jackie1050** [comment]
Phil (the accountant) replied:  

It is not possible to convert a company limited by shares to one limited by guarantee. 

The only way to do this would be to set up a new CIC limited by guarantee and close the old company down.

New bank accounts would need to be set up and any leases etc. would need to be transferred.

He would expect the cost of forming the new company and closing the old one to be £300 plus VAT. We would need to deal with the admin issues - opening new bank acccount etc.

He didn't answer the question about what we would have to tell our members (probably because obvious that we'd have to get them to join a new organisation, set up new standing orders etc.)

**Comment by DoESsean** [comment]
@Sean-anotherone

**Comment by amcewen** [comment]
https://www.communitycompanies.co.uk/converting-company-limited-by-shares-to-company-limited-by-guarantee has some more notes about the process of switching: usually it makes sense to switch at the end of a financial year, and there might be an implication for directors as it would be seen as a sale of the company (I think?  I haven't checked that bit in detail)

**Comment by Sean-anotherone** [comment]
I seem to have been copied in on this, probably because I commented previously...
It's a simplistic view, but;
No individual or group of individuals make any profit from DoESDoes provides a service to the community (local, as well as small startup and maker communities)Much of that service is available for free, at least some of the timeThere are things you charge for, and that money goes into paying the bills & making the service better.The stuff Does "does" attracts support in the form of equipment donations, and the like
Sounds like a charity to me.
The savings on rates alone would likely offset any aggravation involved. 

Just a 2-penneth.
Sean-anotherone


    On Tuesday, 13 April 2021, 18:02:02 BST, Adrian McEwen ***@***.***> wrote:  
 
 


https://www.communitycompanies.co.uk/converting-company-limited-by-shares-to-company-limited-by-guarantee has some more notes about the process of switching: usually it makes sense to switch at the end of a financial year, and there might be an implication for directors as it would be seen as a sale of the company (I think? I haven't checked that bit in detail)

**Comment by seanspotatobusiness** [comment]
Edinburgh Hacklab is set up as a charity. I don't know anything about
the relative benefits of different structures though.

On Tuesday, 13 Apr, 20:25,  wrote:

> I seem to have been copied in on this, probably because I commented previously...
 It's a simplistic view, but;
 No individual or group of individuals make any profit from DoESDoes
provides a service to the community (local, as well as small startup
and maker communities)Much of that service is available for free, at
least some of the timeThere are things you charge for, and that money
goes into paying the bills &amp; making the service better.The stuff
Does "does" attracts support in the form of equipment donations, and
the like
 Sounds like a charity to me.
 The savings on rates alone would likely offset any aggravation involved.

 Just a 2-penneth.
 Sean-anotherone


     On Tuesday, 13 April 2021, 18:02:02 BST, Adrian McEwen
***@***.***&gt; wrote:




 https://www.communitycompanies.co.uk/converting-company-limited-by-shares-to-company-limited-by-guarantee
has some more notes about the process of switching: usually it makes
sense to switch at the end of a financial year, and there might be an
implication for directors as it would be seen as a sale of the company
(I think? I haven't checked that bit in detail)

**Comment by amcewen** [comment]
I don't think we're planning to do anything on this anytime soon, but I ended up [blogging a bit (tangentially) about it](http://www.mcqn.net/mcfilter/archives/thinking/what_if_cooperatives_arent_the_answer.html), so just logging that here for future reference.
